ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -100,7 +100,7 @@ type structure_interface = {
     "driver"  ? string
     "bootproto" ? string
     "onboot" ? string
-    "type"    ? string with match(SELF, '^(Ethernet|Bridge|Tap|xDSL)$')
+    "type"    ? string with match(SELF, '^(Ethernet|Bridge|Tap|xDSL|OVSBridge|OVSPort)$')
```

Comment:
@Pansanel maybe you should also include OVS_EXTRA to configure extra OVS options for bridges and ports, in some cases this option is quite useful (to set the VLAN trunk for example), I found this from ovs github doc: 
http://git.openvswitch.org/cgi-bin/gitweb.cgi?p=openvswitch;a=blob_plain;f=rhel/README.RHEL;hb=HEAD